Resources/Locale/en-US/prototypes/roles/antags.ftl

```diff
@@ -31,5 +31,5 @@ roles-antag-space-ninja-objective = Use your stealth to sabotage the station, no
 roles-antag-thief-name = Thief
 roles-antag-thief-objective = Add some NT property to your personal collection without using violence.
 
-roles-antag-terminator-name = Exterminator
-roles-antag-terminator-objective = Kill the target at all costs, the future depends on it.
+roles-antag-terminator-name = Paradox Anomaly # DeltaV - paradox anomaly
+roles-antag-terminator-objective = Replace your double, or befriend them. # DeltaV - paradox anomaly
```

Comment:
Is this valid Fluent syntax? Its not mentioned in the [docs](https://projectfluent.org/fluent/guide/comments.html)

Reply:
it does work but only for singleline
if you do it for multiline the comment is included

very funny parser